pkg/projectfile/projectfile.go

```diff
@@ -822,17 +822,20 @@ func GetProjectFilePath() (string, *failures.Failure) {
 		getProjectFilePathFromWd,
 		getProjectFilePathFromDefault,
 	}
+	var path string
+	var fail *failures.Failure
 	for _, getProjectFilePath := range lookup {
-		path, fail := getProjectFilePath()
+		path, fail = getProjectFilePath()
 		if fail != nil {
-			return "", fail
+			logging.Debug("Failed to find project file path: %v", fail)
 		}
+
 		if path != "" {
 			return path, nil
 		}
 	}
 
-	return "", FailNoProject.New(locale.T("err_no_projectfile"))
+	return "", FailNoProject.Wrap(fail, locale.T("err_no_projectfile"))
```

Comment:
This would cause red herrings. The failure you're wrapping is not what caused the error.